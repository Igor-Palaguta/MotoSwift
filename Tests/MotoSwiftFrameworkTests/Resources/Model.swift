import Foundation

enum Entities: String {
   case AllTypes
   case NumericTypes
   case ScalarTypes
}
