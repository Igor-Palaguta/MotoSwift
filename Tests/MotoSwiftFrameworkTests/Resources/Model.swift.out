// DO NOT EDIT. This file is machine-generated and constantly overwritten.
// Make changes to model.stencil instead.

import Foundation

// swiftlint:disable file_length
// swiftlint:disable type_body_length
// swiftlint:disable nesting
public struct CoreDataEntity {
   public struct AllTypes {
      public static let name = "AllTypes"

      public class Field: NumericTypes.Field {
         public static let data = "data"
         public static let date = "date"
         public static let string = "string"
         public static let transformable = "transformable"
      }
   }
   public struct NumericTypes {
      public static let name = "NumericTypes"

      public class Field {
         public static let boolean = "boolean"
         public static let decimal = "decimal"
         public static let double = "double"
         public static let float = "float"
         public static let int16 = "int16"
         public static let int32 = "int32"
         public static let int64 = "int64"
         public static let scalars = "scalars"
      }
   }
   public struct ScalarTypes {
      public static let name = "ScalarTypes"

      public class Field {
         public static let boolean = "boolean"
         public static let double = "double"
         public static let float = "float"
         public static let int16 = "int16"
         public static let int32 = "int32"
         public static let int64 = "int64"
         public static let numerics = "numerics"
         public static let eq_true = "eq_true"
         public static let gt_100 = "gt_100"
      }
   }
}
